main.go

```diff
@@ -22,21 +22,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	if os.Getenv("AWS_REGION") == "" {
-		fmt.Printf("AWS_REGION not set")
-		os.Exit(1)
-	}
-
-	region := os.Getenv("AWS_REGION")
-
 	log.Printf("source queue : %v", *src)
 	log.Printf("destination queue : %v", *dest)
 
-	config := &aws.Config{
-		Region: &region,
-	}
-
-	client := sqs.New(session.New(), config)
+	session := session.Must(
```

Comment:
I checked the [source code](https://github.com/aws/aws-sdk-go/blob/master/aws/session/session.go#L297-L303) and it just checks the `err` and panics if it is set. Since this is exactly what I would do as a replacement (like in line 49), what's the point in it? Or what else do you suggest?